### PR TITLE
credit cards (detail-list) almost done, reusable confirmation/message template and more

### PR DESCRIPTION
**Pull request by ronmelcuba10**

### CODE REVIEW

File: books/urls.py [hunk 1]
```diff
@@ -8,13 +8,7 @@
     url(r'^$', views.index, name='index'),
 
     # /books/05/ this will show a page with the details od the views
-<<<<<<< HEAD
     url(r'^(?P<book_id>[0-9]+)/$', views.book_detail, name='detail'),
-
-    # /author/05/ this will show a page with the list of author books views
-    url(r'^author/(?P<author_id>[0-9]+)/$', views.author_list, name='author_books'),
-=======
     url(r'(?P<book_id>[0-9]+)/$', views.book_detail, name='detail'),
```
Comment (jalvarado91) [inline books/urls.py:12]: Looks like this is here twice now

File: creditcards/templates/partials/_creditcards_list.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{% load bootstrap3 %}
+{% block content %}
+<table>
+    <tbody>
+        {% for creditcard in creditcards_list %}
+            <tr>
+                <td>
+                    <a href="{% url 'users:creditcardbound' user.id creditcard.id %}">{{ creditcard.name }}&nbsp&nbsp</a>
+                </td>
```
Comment (jalvarado91) [inline creditcards/templates/partials/_creditcards_list.html:9]: `&nbsp&nbsp` missing semicolons. Should be ``&nbsp;&nbsp;`

File: users/admin.py [hunk 3]
```diff
@@ -17,6 +17,7 @@ class UserAdmin(BaseUserAdmin):
...
 
 
-# Re-register UserAdmin
-admin.site.unregister(User)
-admin.site.register(User, UserAdmin)
\ No newline at end of file
+admin.site.register(Profile)
+"""
+admin.site.register(User, UserAdmin)
+"""
```
Comment (jalvarado91) [inline users/admin.py:23]: We did we comment this out? This was working now.

File: users/models.py [hunk 4]
```diff
@@ -7,5 +7,20 @@
...
+
+def create_profile(sender, **kwargs):
+    user = kwargs["instance"]
+    if kwargs["created"]:
+        profile = Profile(user=user)
+        profile.nick_name = ''
+        profile.save()
+
+
+post_save.connect(create_profile, sender=User)
```
Comment (jalvarado91) [inline users/models.py:26]: I don't think we need this. I had done fixes that already showed the profile fields under the user in the admin section.

File: creditcards/templates/partials/_creditcards_list.html (hunk 2 above)
Comment (ronmelcuba10) [reply creditcards/templates/partials/_creditcards_list.html:9]: fixed

File: users/models.py (hunk 4 above)
Comment (ronmelcuba10) [reply users/models.py:26]: In the repo's version, when you create a superuser it does not create a profile( One ---> Zero or One). With these changes, there is always a profile for every user( One ---> One)

**Pull request closed without merging**